core/templates/dev/head/pages/exploration-editor-page/editor-tab/state-name-editor/state-name-editor.directive.ts

```diff
@@ -56,29 +56,32 @@ angular.module('oppia').directive('stateNameEditor', [
             StateEditorService, StateNameService, FocusManagerService,
             ExplorationStatesService, RouterService) {
           var ctrl = this;
+          StateNameService.init();
           ctrl.EditabilityService = EditabilityService;
           ctrl.StateEditorService = StateEditorService;
           ctrl.StateNameService = StateNameService;
-          StateNameService.setStateNameEditorVisibility(false);
```

Comment:
Shouldn't we still be setting the visiibility to false at the outset?

Reply:
`StateNameService.init()` does that (called above).